frontend/src/screens/wallet/WithdrawOnchainFunds.tsx

```diff
@@ -159,6 +173,34 @@ export default function WithdrawOnchainFunds() {
           and have the seed phrase for.
         </p>
         <form onSubmit={redeemFunds} className="grid gap-5 mt-4">
+          <div className="">
+            <Label htmlFor="amount">Amount</Label>
+            <div className="flex justify-between items-center mb-1">
+              <p className="text-sm text-muted-foreground">
+                Current onchain balance:{" "}
+                {new Intl.NumberFormat().format(balances.onchain.total)} sats
+              </p>
+              <div className="flex items-center gap-1">
+                <Checkbox
+                  id="send-all"
+                  onCheckedChange={() => setSendAll(!sendAll)}
+                />
+                <Label htmlFor="send-all" className="text-xs">
+                  Send All
+                </Label>
+              </div>
+            </div>
+            <Input
+              id="amount"
+              type="number"
+              value={sendAll ? balances.onchain.total : amount}
```

Comment:
are we sure "send all" will be that amount?

Reply:
No, good catch!